clients/java/client/src/test/java/org/camunda/bpm/client/impl/ExternalTaskClientBuilderImplTest.java

```diff
@@ -91,4 +94,17 @@ public String getBaseUrl() {
     }
   }
 
+  @Test
+  public void testWithNullBaseUrl() {
```

Comment:
My understanding of the scenario is that with this fix we're just avoiding an NPE specifically when trying to access the baseURL without first calling `.build`.  To differentiate this test case from the ones in `ClientIT`, can we go for a name that makes this more clear? 
Maybe something like `testNoNPEWhenRetrievingNullBaseUrlWithoutBuilding` or similar 🤔  ..bit verbose.
And then in the assertions we can probably omit the assertion on the exception upon build since thats already covered (see other comment)